docs/source/release/v6.4.0/direct_geometry.rst

```diff
@@ -59,18 +59,17 @@ MSlice
 New Features
 ############
 
-- Various documentation updates
-- MSlice now available as a noarch conda package
-- Update to matplotlib 3.5
-- Add ability to change slice plot font sizes using quick options
-- Changed default cut algorithm to Integration
+- The documentation has been improved.
+- The programme is now available as a noarch conda package.
+- MSlice has been updated to matplotlib 3.5.
```

Comment:
Can we change this to Mslice is now compatible with ``matplotlib v3.5.0``. ? It makes it more consistent with the Workbench release notes.